src/components/vr_cooperation/src/command/support_service_request.cc

```diff
@@ -55,24 +55,21 @@ void SupportServiceRequest::Execute() {
   message_.set_rpc_type(vr_hmi_api::REQUEST);
   message_.set_correlation_id(parent()->GetNextCorrelationID());
   parent()->SendMessageToHMI(message_);
+  parent()->UnregisterRequest(message_.correlation_id());
```

Comment:
I'm not sure that unregistering request should be here because this action deletes the command and you never do receive response.